src/components/todo-item.vue

```diff
@@ -1,7 +1,20 @@
 <template>
   <li>
-    <label>{{todo}}</label>
-    <button @click="$emit('removeTodoItem')">Remove</button>
+    <div v-if="!todo.editing" @click="editMode(todo)">
```

Comment:
u can use `isEditing` condition without item attribute (it is redundant)

Reply:
Fixed.